.github/workflows/pr-export.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint, format and test exports
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "export/**"
+      - ".github/workflows/pr-export.yml"
```

Comment:
What is the meaning of 'PR' in 'PR-export'?

Reply:
It's for `pull request` since my plan was to run these tests on all export job PRs.  I'll adjust the workflow name to make things clearer.